@navikt/core/react/src/modal/ModalUtils.ts

```diff
@@ -1,6 +1,20 @@
 import React from "react";
 import type { ModalProps } from "./types";
 
+export interface MouseCoordinates {
+  clientX: number;
+  clientY: number;
+}
+
+export const coordsAreInside = (
+  { clientX, clientY }: MouseCoordinates,
+  { x, y, width, height }: DOMRect,
+) => {
+  if (clientX < x || clientY < y) return false;
+  if (clientX > x + width || clientY > y + height) return false;
```

Comment:
Har sett litt på [hva DOMRect inneholder](https://developer.mozilla.org/en-US/docs/Web/API/DOMRectReadOnly), og det ser ut som man kan bruke `right` og `bottom` i stedet for hhv. `x + width` og `y + height`? Har ikke testet det selv. I så fall kan vi også bruke tilsvarende `top` og `left` i stedet for `x` og `y` for bedre lesbarhet.